### ISSUE TITLE
Disable ticking sound during breaks

### ISSUE THREAD

**Issue by ccidral** [tomighty/tomighty#25]
As discussed in: http://groups.google.com/group/tomighty/browse_thread/thread/117a960fa5c83bf8

Add an option to disable the ticking sound during breaks.

**Comment by AdrianoFerrari** [comment]
I agree. Absolutely essential!

**Comment by agileapricot** [comment]
Yes please!

It's stressing me :)

**Comment by ccidral** [comment]
Moving to milestone 0.8

**Comment by AdrianoFerrari** [comment]
Thanks ccidral!

This is already one of the best Pomodoro timers out there, and this one feature will push it to #1

**Comment by ccidral** [comment]
Hey thanks, you're welcome.

I hope to have some spare time next week to work on Tomighty in order to get the 0.8 version out as soon as possible.

**Comment by stanleywinston** [comment]
Is this still coming? Would make this the perfect pomodoro app IMO, the ticking noise stresses me out during breaks though so I'm turning it off and using a separate timer.

**Comment by CrazyPyro** [comment]
This should be marked as complete.  I'm running 0.8, and in the Options, there's a Sounds tab with an "Enable tic-tac" box that you can un-check.

**Comment by ccidral** [comment]
@stanleywinston Yes, it is. I have plans to scrap the Java version of Tomighty and replace it by a .NET version, so it might take a while.

**Comment by CrazyPyro** [comment]
@ccidral, waitwhat - a .NET rewrite?  Why?  (Sorry if there's a better place to discuss this - just let me know.)

As far as this issue, goes, shouldn't it be closed per my previous comment?  I see there's a new issue #95 that might help eliminate any other confusion on this.

**Comment by ccidral** [comment]
@CrazyPyro Some aspects of Java's GUI toolkit don't behave well in some platforms.  For example, the system tray is not supported in some desktop environments (see #48, #50 and #52).  Also in some desktop envs the system tray icon lacks support for transparent background.

I want to write native versions for each major operating system (Linux, OSX and Windows) to eliminate those problems.  The OSX version is done.  The Windows (.NET) version will hopefully come soon (right @dcidral ?).  For Linux, I'm considering writing a native version in Vala/GTK+, or perhaps Qt.

**Comment by CrazyPyro** [comment]
Hmm, I can appreciate the tension and trade-off between cross-platform portability vs native feel.  Personally, I lean heavily in favor of the former at the expense of the latter, but I understand many would feel the exact opposite.  But it sure is a shame to write the same thing 3 times (at least).  Especially since it seems like 90% of the problems are related to the System Tray, and that those are more the fault of the Swing GUI rather than the Java language itself.  Perhaps something like [JavaFX](http://docs.oracle.com/javafx/2/overview/jfxpub-overview.htm#A1095238) instead?

I'm not sure what the advantage of a C# port would be, since the Java version seems to work just fine on Windows (7 at least).  Unless Mono+C#+GTK# was going to be the new single cross-platform solution? (Although I don't think GTK is going to be any more satisfying in that role than Swing.)  If you really do still want to do a C# version, I'd say convert it automatically with something like [Sharpen](http://community.versant.com/Projects/html/projectspaces/db4o_product_design/sharpen.html) the way [NGit](https://github.com/mono/ngit) did. (Although you'd then still have to manually rewrite the GUI calls.)

Still, what happens with plugins?  Do we then have to write our plugin 3 times as well...?  Just some more stuff to think about...

**Comment by leonardiwagner** [comment]
I think that ticking sound should be disabled by default. I'm pretty sure that a lot of people doesn't use tomighty due to ticking sound as default.

Also, ticking sound is useless, plus the 25 min is all about being focused in a task.. and ticking sound is against that!

**Comment by quinnfive** [comment]
4+ years later and I'm afraid the ticking sound is still not able to be disabled during the breaks... kinda sucks. 

Any plans on continuing to work on this app at all? 

I really like Tomighty - so many features about it that are great and there are very few half decent pomodoro apps out there...

**Comment by h3x0v3rl0rd** [comment]
update the `config.el`

(after! pomidor
  (setq pomidor-play-tick nil
        pomidor-sound-tick nil
        pomidor-sound-tack nil
        pomidor-sound-overwork nil))


After modifying your config:

- Save the file (C-x C-s).
- Restart Emacs or reload the config (M-x doom/reload).
